### DISCUSSION TITLE
Email configuration on Heroku

### DISCUSSION THREAD

**Discussion by dvb7836** [saleor/saleor#6007] [Deployment]
Saleor API and dashboard deployed on Heroku and EMAIL_URL added to environment variables. My email provider for now is Mailgun. I can send email with curl but when adding user to dashboard I get "ERROR". No logs or details about this error in container logs.
Can you suggest where I can find logs about email sending errors or what should I do to configure it properly?

Thank you

**Comment by karolkielecki** [comment]
Hey @dvb7836. What is your current version of Saleor? Did you follow our docs? https://docs.saleor.io/docs/2.9.0/guides/email-integration/#mailgun-integration
